### DISCUSSION TITLE
Import from Tessie

### DISCUSSION THREAD

**Discussion by johnnyoshika** [teslamate-org/teslamate#3154] [Q&A]
Has anyone successfully imported data from Tessie to TeslaMate? I asked [here](https://teslamotorsclub.com/tmc/threads/export-data-from-tessie-and-import-into-teslamate.296401/) and also asked Tessie's developer. The only suggestion I got was to export a CSV from Tessie (which I did) and map it back to TeslaFi's format to import into TeslaMate. I tried to do the mapping between Tessie's CSV and TeslaFi's but it's hard to know exactly how all the columns should be mapped.

**Comment by malekcellier** [comment]
Any progress on that? I am also interested

Cheers

**Comment by johnnyoshika** [comment]
Nope. Let me know if you make any progress. I still have a lot of data in Tessie that I'd like to import into TeslaMate.

**Reply by parkr** [reply]
Perhaps you could include an example of all of the columns Tessie exports? And what the export is (charges vs drives etc). Then folks might be able to help map. As it currently stands, no one on this forum has any info about the input data Tessie provides.

**Reply by DrMichael** [reply]
An export is here: https://github.com/teslamate-org/teslamate/discussions/4112#discussioncomment-10345665

**Comment by h1ghland3r** [comment]
Hello guys. Have anyone figured out how to do it?
